README.Rmd

```diff
@@ -14,6 +14,8 @@ knitr::opts_chunk$set(
 
 # deparse
 
+[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/NikNakk/deparse?branch=master&svg=true)](https://ci.appveyor.com/project/NikNakk/deparse)
```

Comment:
Does this need to point to the source repo? Also, could you please add the Travis badge?